Cooper/ClientApp/src/app/sign-in/sign-in.component.ts

```diff
@@ -17,7 +17,15 @@ export class SignInComponent {
 
   invalidLogin: boolean;
 
-  constructor(private router: Router, private http: HttpClient) { }
+  constructor(private router: Router, private http: HttpClient) {
+    this.CheckAuthentification();
+  }
+
+  CheckAuthentification(): void {
+    var Token = localStorage.getItem("JwtCooper");
```

Comment:
if you install plugin TSLint, it will check your code for codestyle.
If value won't be changed - use const Token = .......; Optimization.